src/main/java/multuthreadingAndBlockingQueue/Main.java

```diff
@@ -0,0 +1,91 @@
+
+
+public class Main {
+    public static void main(String[] args) throws Exception {
+        System.out.println("Synchronized and volatile examples");
+
+        Account account = new Account(100, 100, new String[]{"Petrov", "Ivan", "Sydorovych"});
+        System.out.println("Initial dollar balance = " + account.getDollarBalance());
+        System.out.println("Initial hryvnya balance = " + account.getHryvnyaBalance());
+
+
+        Runnable withdrawObj = new Runnable() {
+            public void run() {
+                for (int i = 0; i < 5; i++) {
+                    System.out.println("Dollar balance before withdrawal = " + account.getDollarBalance());
+                    account.withdrawDollars(10);
+                    System.out.println("Dollar balance after withdrawal = " + account.getDollarBalance());
+                }
+            }
+        };
+
+        Runnable depositObj = new Runnable() {
+            public void run() {
+                for (int i = 0; i < 5; i++) {
+                    System.out.println("Dollar balance before deposit = " + account.getDollarBalance());
+                    account.depositDollars(10);
+                    System.out.println("Dollar balance after deposit = " + account.getDollarBalance());
+                }
+            }
+        };
+        Runnable withdrawObj2 = new Runnable() {
+            public void run() {
+                for (int i = 0; i < 5; i++) {
+                    System.out.println("Hryvnya balance after withdrawal = " + account.getHryvnyaBalance());
+                    account.withdrawHryvhya(10);
+                    System.out.println("Hryvnya balance after withdrawal = " + account.getHryvnyaBalance());
+
+                }
+            }
+        };
+
+        Runnable depositObj2 = new Runnable() {
+            public void run() {
+                for (int i = 0; i < 5; i++) {
+                    System.out.println("Hryvnya balance before deposit = " + account.getHryvnyaBalance());
+                    account.depositHryvhya(10);
+                    System.out.println("Hryvnya balance after deposit = " + account.getHryvnyaBalance());
+                }
+            }
+        };
+        Thread depositThread = new Thread(depositObj);
+        Thread withdrawalThread = new Thread(withdrawObj);
+
+
+        Thread depositThread2 = new Thread(depositObj2);
+        Thread withdrawalThread2 = new Thread(withdrawObj2);
+
+        depositThread.start();
+        withdrawalThread.start();
+
+        try {
+            Thread.sleep(10000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+
+        depositThread2.start();
+        withdrawalThread2.start();
+
+        try {
+            Thread.sleep(4000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+
+        System.out.println("BlockingQueue examples");
+        BlockingQueue<Integer> bq = new LinkedBlockingQueue<>(1);
```

Comment:
For one element present special collections

Reply:
OK, I'll make changes. I just wanted to examine how method take removes elements)